src/routes/safe/components/Transactions/TxList/TxDataRow.tsx

```diff
@@ -46,7 +47,7 @@ const generateInlineTypeValue = (
           hash={value}
           shortenHash={8}
           showCopyBtn
-          explorerUrl={hasExplorer ? getExplorerInfo(value) : undefined}
+          explorerUrl={getExplorerInfo(value)}
```

Comment:
What about if it doesn't have an explorer?

Reply:
I checked that we were not passing the `hasExplorer` (which defaults to true) prop for this `inlineType` thus it's safe to remove here